io/shared/src/test/scala/fs2/io/file/PathSuite.scala

```diff
@@ -44,6 +44,20 @@ class PathSuite extends Fs2IoSuite {
   test("construction") {
     assertEquals(Path("foo/bar"), Path("foo") / "bar")
     assertEquals(Path("/foo/bar"), Path("/foo") / "bar")
+    forAll((sepLength: Int) =>
+      if (sepLength >= 0 && sepLength <= 100)
```

Comment:
These `if` guards are probably skipping most/all of the generated values. Instead you should do something like this.
```suggestion
    forAll(Gen.choose(0, 100))((sepLength: Int) =>
```